### PR TITLE
Update README.md file with link pointing to my deployed application -…

### PR DESCRIPTION
**Pull request by ogagaoghene**
# Application deploy

## Links 
https://ogagaoghene.github.io/v1-mobile-web-app/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ Portfolio: Mobile Version
 v2-desktop-version-with-animations
 Request this url: https://github.com/ogagaoghene/v1-desktop-version-with-animations in your web browser. Once opened navigate to the top left level of the project a green code download button will be visible on the righthand side. Select download Zip option from drop down menu. Once the download is complete you will be able to access my project locally.
 
-## Links 
- https://ogagaoghene.github.io/v1-mobile-web-app/
- 
+## Links (GitHub Page link for application deploy - v1-desktop-version-with-animations)
+https://ogagaoghene.github.io/v1-mobile-web-app/
+
```
Comment (omar-muhamad) [inline README.md:28]: - [x] [Optional] It would be better professionally if you add this link in the` live demo` section above in the Readme file.

**Review by omar-muhamad — APPROVED** [review]
# Approved ✔
### Hi @ogagaoghene,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- Please check the inline comments below ⬇.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by ogagaoghene** [comment]
Thanks for your professional review and suggestion. I appreciate and understand your point. It is a valid one.

File: README.md (hunk 1 above)
Comment (ogagaoghene) [reply README.md:28]: I agree and will do that subsequently. Thanks

**Pull request merged**